tqdm/_tqdm.py

```diff
@@ -944,8 +945,10 @@ def set_postfix(self, **kwargs):
             # Else for any other type, try to get the string conversion
             elif not isinstance(kwargs[key], basestring):
                 kwargs[key] = str(kwargs[key])
+            # Else if it's a string, don't need to preprocess anything
```

Comment:
This is intended like this to explicitly explain that we don't preprocess strings. This comment can be removed as it can be deduced by other conditions, but I like to be explicit.